modules/stochastic_tools/doc/content/modules/stochastic_tools/install_pytorch.md

```diff
@@ -0,0 +1,146 @@
+# Install MOOSE with Pytorch C++ API (libtorch)
+
+The way one enables pytorch capabilities in MOOSE depends on
+the operating system (Linux or Mac) and if we use HPC or just a local workstation.
+
+!alert! note
+Before we review the main approaches, it is important to emphasize that
+linking MOOSE with libtorch on Linux machines is not support if the compiler stack has been built
+using a `libc` version below 2.27 (for `libtorch v 1.8+`)
+or 2.23 (for `libtorch v1.4-1.8`). Furthermore, we do not support `libtorch` versions below
+v1.4. To check your currently used libc version use the following command:
+
+```bash
+ldd --version
+```
+
+!alert-end!
+
+## Installation on a local Mac workstation
+
+The first step of this process is to set up the a suitable conda environment. For this follow
+the instructions [here](https://mooseframework.inl.gov/getting_started/installation/conda.html).
+Next, activate the conda environment (assume `moose-torch`), clone the repository
+ and navigate to the base folder of the
+stochastic tools module (let's say it is in `~/projects`):
+
+```bash
+conda activate moose-torch
+git clone https://github.com/grmnptr/moose.git moose-torch
+cd moose-torch
+git checkout torch-compile-19571
+cd ~/projects/moose/modules/stochastic_tools
```

Comment:
Do I have to have conda? If I can build pytorch from scratch, anyway I can link my pytorch?

Reply:
No, you don't need conda. You can specify a `TORCH_DIR="path to your installation"` and libtorch.so/dylib has to be in the `$TORCH_DIR/lib` folder. So we will look for: `$TORCH_DIR/lib/libtorch.sh/dylib`. The includes need to be in the `$(TORCH_DIR)/include/torch/csrc/api/include` folder.